src/components/SummaryBar/SummaryBar.jsx

```diff
@@ -24,31 +24,26 @@ import badges_icon from './badges_icon.png';
 import bluesquare_icon from './bluesquare_icon.png';
 import report_icon from './report_icon.png';
 import httpService from '../../services/httpService';
-import { faWindowMinimize } from '@fortawesome/free-regular-svg-icons';
 import { ENDPOINTS } from 'utils/URL';
 import axios from 'axios';
 import { ApiEndpoint } from 'utils/URL';
-import hasPermission from 'utils/permissions';
 import { getProgressColor, getProgressValue } from '../../utils/effortColors';
 
 const SummaryBar = props => {
   const { asUser, role, leaderData } = props;
   const [userProfile, setUserProfile] = useState(undefined);
+  const [infringements, setInfringements] = useState(0);
+  const [badges, setBadges] = useState(0);
+  const [totalEffort, setTotalEffort] = useState(0);
+  const [weeklySummary, setWeeklySummary] = useState([]);
+
   const [tasks, setTasks] = useState(undefined);
   const authenticateUser = useSelector(state => state.auth.user);
+  const gsUserprofile = useSelector(state => state.userProfile);
+  const gsUserTasks = useSelector(state => state.userTask);
```

Comment:
@RJBornToWin You think is possible to rename these two "gs" variables to improve readability?